app/forms/EmployerContributionFormProvider.scala

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package forms
+
+import forms.mappings.Mappings
+import javax.inject.Inject
+import play.api.data.Form
+
+class EmployerContributionFormProvider @Inject() extends Mappings {
+
+  def apply(): Form[Int] =
+    Form(
+      "value" -> int(
+        "employerContribution.error.required",
+        "employerContribution.error.wholeNumber",
+        "employerContribution.error.nonNumeric")
+          .verifying(inRange(0, 2500, "employerContribution.error.outOfRange"))
```

Comment:
this should be .verifying(inRange(0, Int.MaxValue, "employerContribution.error.outOfRange"))